### ISSUE TITLE
 mikrotik container

### ISSUE THREAD

**Issue by mrdnrd68** [openwrt/docker#102]
hi
how to deploy openwrt in
I import this image in mikrotik container but mikrotik running then immediately stop
plz help me

**Comment by aparcar** [comment]
Where do you run this? Mikrotik?

**Comment by mrdnrd68** [comment]
yes on mikrotk install in vm x86-64 i can run pihole but i import openwrt
successfuly when start container mikrotik stop this
plz help how to run openwrt on mikrotik
thanks

**Comment by mrdnrd68** [comment]
on mikrotin container plz help me

**Comment by mrdnrd68** [comment]
![Uploading IMG_20230317_011727.jpg…]()

**Comment by mrdnrd68** [comment]
![Uploading IMG_20230317_011727.jpg…]()

**Comment by mrdnrd68** [comment]
help me
